### PR TITLE
Columns summary

### PR DESCRIPTION
**Pull request by clementdevos**
Adds ColumnSummary in renderer() function : 
Needed to compute a share on the fly from the cellValue/columnSum

Working in examples, not unitTested

### CODE REVIEW

File: src/Utils.js [hunk 1]
```diff
@@ -276,7 +276,7 @@ export function renderCell(cellData, column, rowData, rowNumber, tableData, colu
 
     // render normally if a renderer is provided
     } else if (renderer != null) {
-      return renderer(cellData, { column, rowData, rowNumber, tableData, columns });
+      return renderer(cellData, { column, rowData, rowNumber, tableData, columns ,columnSummary });
```
Comment (pbeshai) [inline src/Utils.js]: Can you fix this comma

File: src/TacoTableCell.js [hunk 2]
```diff
@@ -162,12 +162,12 @@ class TacoTableCell extends React.Component {
...
   render() {
     const { column, rowData, rowNumber, tableData, columns,
-      onHighlight, highlightedColumn, columnGroup, isBottomData } = this.props;
+      onHighlight, highlightedColumn, columnGroup, isBottomData, columnSummary } = this.props;
     const { className, type } = column;
 
     const cellData = getCellData(column, rowData, rowNumber, tableData, columns, isBottomData);
 
-    const rendered = renderCell(cellData, column, rowData, rowNumber, tableData, columns, isBottomData);
+    const rendered = renderCell(cellData, column, rowData, rowNumber, tableData, columns, isBottomData,columnSummary );
```
Comment (pbeshai) [inline src/TacoTableCell.js]: Can you fix the formatting here - add a space after the comma

File: package.json [hunk 3]
```diff
@@ -74,7 +74,7 @@
     "mocha": "^3.0.2",
     "node-sass": "^3.7.0",
     "null-loader": "^0.1.1",
-    "pbeshai/minami": "^1.1.1",
+    "minami": "pbeshai/minami",
```
Comment (pbeshai) [inline package.json:77]: Thanks for fixing this. Oops.

**Review by pbeshai — CHANGES_REQUESTED** [review]
Good idea!

File: src/TacoTableCell.js (hunk 2 above)
Comment (clementdevos) [reply src/TacoTableCell.js]: Done in #5f3c007

File: src/Utils.js (hunk 1 above)
Comment (clementdevos) [reply src/Utils.js]: Done in #5f3c007

**Review by pbeshai — APPROVED** [review]

**Comment by pbeshai** [comment]
Updated docs in #42. Published to npm as 0.4.2. Thanks!

**Comment by clementdevos** [comment]
:+1:

**Comment by clementdevos** [comment]
Hey, did you update the lib folder before publishing? 

Or was it my job? 

image

image

Cheers !

**Comment by pbeshai** [comment]
Ugh, I really need to update the build process here. Sorry about that. 0.4.3 should have it.

**Pull request merged**